test/ngAnimate/animateCssSpec.js

```diff
@@ -1234,6 +1234,67 @@ describe("ngAnimate $animateCss", function() {
             $timeout.flush();
           }).not.toThrow();
         }));
+
+        it("should consider a positive options.delay value for the closing timeout",
```

Comment:
I'm not sure why this test fails. It should ensure that the animation ends exactly then when it's supposed to be, not before.